tools/ReadConsoleInputStream/ConcurrentCircularQueue.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+
+namespace Nivot.Terminal
```

Comment:
>Nivot [](start = 10, length = 5)

Might want to drop this particular namespace for putting it into our Microsoft.console repo.

Reply:
What namespace would you like?

Reply:
DIdn't get that far mentally. Demo.Terminal? Example.Terminal? I don't know. I don't really mind if it stays associated with you, but folks have a tendancy to copy/paste.